src/locale/locales/es/messages.po

```diff
@@ -1408,11 +1408,11 @@ msgstr "Cierra el cajón inferior"
 
 #: src/components/dialogs/GifSelect.tsx:276
 msgid "Close dialog"
-msgstr ""
+msgstr "Close dialog"
```

Comment:
`dialog` usually stands for `popup` in most of the context, proper translations should be `ventana emergente`.

Reply:
Replaced only with "Cerrar ventana"